app/controllers/api/v3x0/mixins/index_mixin.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V3x0
+    module Mixins
+      module IndexMixin
+        def index
+          raise_unless_primary_instance_exists
+          render :json => Insights::API::Common::PaginatedResponseV2.new(
+            :base_query => scoped(filtered.where(params_for_list)),
```

Comment:
@abellotti Does this need to inherit from the `V1x0` mixin for the `scoped` method?

Reply:
Nevermind, I see, the controller inheritance is taking care of it.